configure.py

```diff
@@ -356,7 +358,9 @@ def binary(name):
     except:
         pass
     if platform.is_mingw():
-        cflags += ['-D_WIN32_WINNT=0x0601', '-D__USE_MINGW_ANSI_STDIO=1']
+         cflags += ['-D_WIN32_WINNT=0x0601', '-D__USE_MINGW_ANSI_STDIO=1']
```

Comment:
You added whitespace here.

was that intentional?

Reply:
This line was added when I merged master, I can remove the whitespace